ci/cpu/clang16_release.yml

```diff
@@ -0,0 +1,29 @@
+include:
+  - local: 'ci/common-ci.yml'
+
+cpu clang16 release deps:
+  extends: .build_deps_common
+  variables:
+    EXTRA_APTGET: "clang-15" # TODO: clang-16 (not available in 20.04, update to 22.04?)
```

Comment:
In that case it should say "update to something newer than 22.04" :) I just know that clang-16 was not in the default repositories on the current base image and I just wanted to add a todo to remind me later to update it to actually use 16 (15 was sufficient for testing since it has the same issues).